kubernetes/migration/migrate.go

```diff
@@ -229,3 +239,34 @@ func Released(dockerImageID string) bool {
 	}
 	return false
 }
+
+// ValidVersions returns a list of all versions defined
+func ValidVersions() []string {
```

Comment:
This function isn't used currently. 
Will this be used for special cases such as v1.2.3?

Reply:
Yes. 1.2.3 is a valid version - it's a valid _start_ version.  This just returns a simple list of versions that the tool supports, for example, a CLI tool.  It doesn't convey which combinations of versions are valid together.